pkg/cloud/awscloud/awscloud.go

```diff
@@ -296,18 +297,30 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, architectur
 		return "", "", err
 	}
 
+	ec2Tags := []ec2types.Tag{
+		{
+			Key:   aws.String("Name"),
+			Value: aws.String(name),
+		},
+	}
```

Comment:
Here is a small change. Previously the `Name` tag was appended after the `for` loop, now it is above. So that the first tag is always `Name`.

Reply:
Thanks!